r/tests/testthat/helper-expectation.R

```diff
@@ -317,3 +317,7 @@ split_vector_as_list <- function(vec) {
   vec2 <- vec[seq(from = min(length(vec), vec_split + 1), to = length(vec), by = 1)]
   list(vec1, vec2)
 }
+
+err_snap <- function(...) (expect_error(...))
```

Comment:
Please could you add a comment about why these functions are needed?

Reply:
added